esp/platform/application_config.cpp

```diff
@@ -214,6 +219,27 @@ void bindService(IPropertyTree *legacyEsp, IPropertyTree *app, const char *servi
         if (authenticate)
             bindAuthResources(authenticate, app, service, auth);
     }
+
+    // Add contents of applicationBindingInfo to static EspBinding element by service name
+    xpath.setf("applicationBindingInfo/%s", service);
+    IPTree *branch = app->queryBranch(xpath.str());
+    if (branch)
+    {
+        Owned<IAttributeIterator> aiter = branch->getAttributes();
+        ForEach (*aiter)
+        {
+            const char *name = aiter->queryName();
+            bindingEntry->addProp(name+1, aiter->queryValue());
+        }
+
+        xpath.append("/*");
```

Comment:
You already have branch, so don't really need to append to the xpath.
branch->getElements("*"); should be fine.